### PR TITLE
Update CLI and Python Client

### PR DESCRIPTION
**Pull request by deliahu**

### CODE REVIEW

File: python/client/cortex/consts.py [hunk 1]
```diff
@@ -12,8 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Change if PYTHONVERSION changes
-EXPECTED_PYTHON_VERSION = "3.6.9"
```
Comment (RobertLucian) [inline python/client/cortex/consts.py:16]: Nice! We finally get rid of this.

File: python/client/cortex/util.py [hunk 2]
```diff
@@ -26,15 +26,19 @@ def cli_config_dir() -> Path:
 
 
 @contextmanager
-def open_temporarily(path, mode):
-    Path(path).parent.mkdir(parents=True, exist_ok=True)
+def open_temporarily(path, mode, delete_parent: bool = False):
```
Comment (RobertLucian) [inline python/client/cortex/util.py:29]: What do you say about changing `delete_parent` to `delete_parent_if_empty` and then only removing the parent dir if it's empty only?

**Review by RobertLucian — APPROVED** [review]
Generally LGTM!

File: docs/clients/python.md [hunk 3]
```diff
@@ -86,129 +81,39 @@ Delete an environment configured on this machine.
...
   Deployment status, API specification, and endpoint for each API.
 
-## deploy\_realtime\_api
+## deploy\_file
 
 <!-- CORTEX_VERSION_MINOR -->
 
 ```python
- | deploy_realtime_api(api_spec: Dict[str, Any], handler, requirements: Optional[List] = None, conda_packages: Optional[List] = None, force: bool = True, wait: bool = False) -> Dict
+ | deploy_file(config_file: str, force: bool = False, wait: bool = False) -> Dict
```
[the remark this answers is not in the document]
Comment (deliahu) [reply docs/clients/python.md:105]: Agreed!

File: python/client/cortex/client.py [hunk 4]
```diff
@@ -51,365 +45,44 @@ def __init__(self, env: Dict):
     def deploy(
```
[the remark this answers is not in the document]
Comment (deliahu) [reply python/client/cortex/client.py:46]: I didn't understand exactly what you meant for me to test, but I made a note and we'll check it before merging into master

**Pull request merged**